ui/components/multichain/app-header/app-header.scss

```diff
@@ -33,8 +33,8 @@
       max-width: 250px;
     }
 
-    &--avatar-network {
-      padding: 0; // TODO: Remove once https://github.com/MetaMask/metamask-extension/pull/17006 is merged
```

Comment:
Apologies, I closed the PR but did not merge it you may still need this. Once the component-library components are using the new Box component we can remove this

Reply:
Actually now it's merged you could update the Box component in the PickerNetwork to use the component-library version and you could remove this.

Reply:
I am not sure why do we need to update the Picker network here. I don't need the padding so I removed it. CSS update is just to hide the label tag. 🤔